langchain/callbacks/manager.py

```diff
@@ -624,6 +627,91 @@ async def on_tool_error(
         )
 
 
+class CallbackManagerForRetrieverRun(RunManager, RetrieverManagerMixin):
+    """Callback manager for retriever run."""
+
+    def get_child(self) -> CallbackManager:
```

Comment:
We might not need child here, retriever is always leaf?

Reply:
There's at least one case of a retriever wrapping another retriever